.travis.yml

```diff
@@ -7,9 +7,9 @@ script:
 - npm test
 deploy:
   provider: npm
-  email: mcp_infrastructure@cimpress.com
+  email: anf4@cornell.edu
```

Comment:
Uh, why this? Does it have to be?

Reply:
@ryanbreen for visibility.

I turned the cimpress_mcp_infrastructure account into an organization, and added my personal account as a member so I could publish to projects belonging to that organization. I was then able to generate a new API key to publish with.

All that said, you're right it's a little weird it's attached to my name, even if I am the primary contributor. Unfortunately, I don't see a better solution. The alternative is this weird shared account we have that only Zach has access to the email address for, and that has two-factor authentication turned off.

Ideally anyone who is an admin of the organization should be able to generate and delete keys associated with the organization, which doesn't seem possible today (keys have to be generated with a specific account it seems). I'm happy to switch to whatever process we want to use going forward once it's agreed upon (as @zmarois stated to me, he's understandably uncomfortable being a central point of failure in the administration of the account).

Let me know what you guys think.

Reply:
I am uncomfortable being the central point approving this pattern. I am not uncomfortable with my place on the account. Enterprise Solutions can always re-administer the account, and can add more people if they want access.